community/modules/scripts/ramble-execute/variables.tf

```diff
@@ -85,10 +85,33 @@ variable "ramble_runner" {
 }
 
 variable "gcs_bucket_path" {
-  description = <<-EOT
-    The GCS path for storage bucket and the object, starting with `gs://`.
-    If not set, this module will create a new bucket, to isolate its scripts.
-  EOT
+  description = "The GCS path for storage bucket and the object, starting with `gs://`."
+  type        = string
+  default     = null
+}
+
+variable "spack_profile_script_path" {
+  description = "Path to the Spack profile.d script. Created by an instance of spack-setup"
+  type        = string
+  default     = null
+  validation {
+    condition     = var.spack_profile_script_path != null
```

Comment:
I wasn't aware that not defining a default would do the same thing, but yes I wanted an error message that said something like "This can be defined via chaining" but maybe that's better off in the description?

Reply:
I updated the description and used the pattern you mentioned.